2009-04-01  Werner LEMBERG  <wl at gnu.org>

	* tmac/doc.tmac: Call `ec' before mapping characters.

Index: groff/tmac/doc.tmac
diff -u groff/tmac/doc.tmac:1.39 groff/tmac/doc.tmac:1.41
--- groff/tmac/doc.tmac:1.39	Fri Jan  2 23:16:20 2009
+++ groff/tmac/doc.tmac	Mon Oct 26 07:51:02 2009
@@ -6425,6 +6494,9 @@
 .blm doc-empty-line
 .
 .
+.ec
+.
+.
 .\" For UTF-8, map some characters conservatively for the sake
 .\" of easy cut and paste.
 .
@@ -6438,9 +6510,6 @@
 .\}
 .
 .
-.ec
-.
-.
 .\" load local modifications
 .mso mdoc.local
 .
